ChatAPISampleApp/Podfile

```diff
@@ -6,8 +6,8 @@ use_frameworks!
 target 'ZDCChatAPI Sample App' do
   pod 'SDWebImage', '~>3.8'
   pod 'DKImagePickerController'
-  pod 'NSDate+TimeAgo'
-  pod 'ZDCChat/API', '1.3.0.1'
-  pod 'JLToast', '~> 1.4'
+  pod 'DateToolsSwift'
```

Comment:
I know they weren't there but would you mind adding version numbers to these to lock them in?

pod 'DateToolsSwift' ~> '2.0.3'
pod 'DKImagePickerController' ~> '3.5.7'

Are the current versions for each.